src/FormGroup.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {COMMON, get, TYPOGRAPHY} from './constants'
+import theme from './theme'
+import sx from './sx'
+
+const FormGroup = styled.div`
+  margin: ${get('space.3')} 0;
+  font-weight: ${get('fontWeights.normal')};
```

Comment:
It looks like the FormGroup CSS in Primer CSS adds some additional styles to text inputs:
![image](https://user-images.githubusercontent.com/8960591/93915153-70846f00-fcbc-11ea-9f6b-03822458595c.png)

I think we'll want everything above added here - I would probably do this by targeting the `input` as a nested selected here on line 10.

Reply:
It looks like there are also some styles specific to text areas that are included in Primer CSS's form group styles - we should add those here as well!